client/gutenberg/extensions/presets/jetpack/package.json

```diff
@@ -0,0 +1,30 @@
+{
+	"name": "@automattic/jetpack-blocks",
+	"version": "0.1.0-alpha.1",
```

Comment:
Where does this version come from? The only published (as well as `latest` and `next`) is `0.1.0-alpha.30639`.

I want to make sure we can match an up to date version for testing between Jetpack releases.

Reply:
I had set that manually. Will change it to match the published canary.